### PR TITLE
OPTIMADE v1.2.0-rc.1 release prep

### PR DESCRIPTION
**Pull request by ml-evs**
This PR is for preparing the organizational stuff around the v1.2.0-rc.1 release candidate.

- [x] Update schemas and their versions
  Currently I've just bumped the version number, we will need to wait a while for us to catch up with all these features in optimade-python-tools before the schema properly matches the spec.
- [x] Update CHANGELOG
- [x] Bump specification version numbers
- [x] Update contributors list
- [x] OPTIONAL: spruce up the README a bit with logo and updated links
- [x] Bump authors list
- [x] Final spelling audit

### CODE REVIEW

**Comment by ml-evs** [comment]
@rartino this is the final PR required before rc.1, that includes mostly admin changes I made in preparation for 1.2.0, now adjusted for rc.1.

If you are happy to take a look at this before the holidays then I will release once it is merged.

I will also ping @merkys, @vaitkus and @JPBergsma in case they want to run through it, but I will consider your reviews optional :) There will also of course be the final PR that merges the 1.2-rc.1 into `master` as the new "released" version of the specification --- I will make this now on the assumption that this PR gets merged soon.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## v1.2.0-rc.1 (December 2022)
+
+This is the first release candidate of v1.2.0 of the OPTIMADE API specification.
+It should contain all of the new features in the specification, but their
+implementation may be modified in the final release.
+
+Note: The OpenAPI schemas distributed in `./schemas` have not yet been modified with
+the new features 1.2.0.
```
Comment (rartino) [inline CHANGELOG.md:10]:
```suggestion
Note: The OpenAPI schemas distributed in `./schemas` have not yet been modified with the new features 1.2.0.
```
Do we mean to keep to the one sentence per line standard also for this file?

**Review by rartino — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ml-evs) [reply CHANGELOG.md]: Fixed, this was just my editor default

**Comment by ml-evs** [comment]
Thanks for the quick review @rartino! As I said in our final discussion, I will hold off the official release process until tomorrow evening, but I will merge this PR now so I can make the final `develop->master` PR. This could probably go without external review as I will not change anything in `develop`, but I will try to take the time to make sure there is nothing unexpected in it...

All that will then remain for rc.1 is:

- Setting the git tag
- Making the GitHub release (I'll copy the text from the CHANGELOG)

File: AUTHORS [hunk 2]
```diff
@@ -57,6 +60,7 @@ Daniel Speckhard
 Leopold Talirz
 Cormac Toher
 Martin Uhrin
+Antanus Vaitkus
```
Comment (merkys) [inline AUTHORS:63]:
```suggestion
Antanas Vaitkus
```

Comment (ml-evs) [reply AUTHORS:63]: Phew, thanks for catching that @merkys.

**Review by merkys — APPROVED** [review]

**Review by rartino — APPROVED** [review]
Important catch by @merkys ; but now it looks good to me.

**Comment by ml-evs** [comment]
Just a side note that the AUTHORS list still needs to be updated for the main release, but the CECAM website was down so I couldn't grab the attendee list for the last 2 workshops.

File: AUTHORS (hunk 2 above)
Comment (vaitkus) [reply AUTHORS]: Thanks! :smile:

**Comment by JPBergsma** [comment]
The CECAM website is online again. The list of participants can be found at https://www.cecam.org/workshop-details/1120

**Pull request merged**